tests/e2e-openshift/monolithic-multitenancy-openshift/02-assert.yaml

```diff
@@ -11,10 +11,21 @@ apiVersion: v1
 kind: Service
 metadata:
   name: dev-collector
+  namespace: chainsaw-monolithic-multitenancy
 spec:
   ports:
   - appProtocol: grpc
-    name: otlp-grpc
+    name: otlp-grpc-grpc
     port: 4317
     protocol: TCP
-    targetPort: 4317
\ No newline at end of file
+    targetPort: 4317
+  - appProtocol: http
+    name: otlp-http-http
```

Comment:
Can we use `otlp-http` and `otlp-grpc` as port names?

Reply:
These ports names are set by the OTEL operator. They are derived from the receiver name e.g. `otlp/http`

Reply:
oh, sorry, I didn't realize this is from the OTEL collector.